app/policies/upload_policy.rb

```diff
@@ -0,0 +1,5 @@
+class UploadPolicy < ApplicationPolicy
+  def destroy?
+    false
```

Comment:
I feel like we should allow destruction so that, if the user clicks the `(X)` on the upload item in the frontend after it's already uploaded, it'll just immediately delete it instead of waiting to be cleaned out by the automated task